libsplinter/src/network/connection_manager/mod.rs

```diff
@@ -405,17 +398,14 @@ fn send_heartbeats(
     notify_subscribers(subscribers, notifications);
 }
 
-fn create_heartbeat() -> Result<Vec<u8>, ConnectionManagerError> {
-    let heartbeat = NetworkHeartbeat::new().write_to_bytes().map_err(|_| {
-        ConnectionManagerError::HeartbeatError("cannot create NetworkHeartbeat message".to_string())
-    })?;
+fn create_heartbeat() -> Vec<u8> {
+    let heartbeat = NetworkHeartbeat::new().write_to_bytes().unwrap();
```

Comment:
Why is panicking here desirable?

Reply:
I believe @peterschwarz mentioned this a few times, but `write_to_bytes` only fails if the system completely runs out of memory.

Reply:
This deviates from the way we handle these errors everywhere else in splinter, so I'm not sure I 100% agree with this change from a consistency perspective, but I won't block the PR because of that. However, this should be changed to an `expect` to provide an error message.